src/app/utils/accountPageUtils.js

```diff
@@ -77,15 +77,25 @@ const manipulateAccountPage = (
   });
 
   items.forEach((el) => {
-    const locationSelect = el.getElementsByTagName('select')[0];
-    const locationProp = locationSelect ? locationSelect.name : '';
-    let locationValue;
-    el.querySelectorAll('option').forEach((option) => {
-      if (option.selected) locationValue = `${option.value.trim()}+++`;
-    });
-    const locationData = {
-      [locationProp]: locationValue,
-    };
+    const locationData = {};
+    if (contentType === 'holds') {
+      const locationSelect = el.getElementsByTagName('select')[0];
+      if (!locationSelect) return;
+      const locationProp = locationSelect.name;
+      let locationValue;
+      el.querySelectorAll('option').forEach((option) => {
+        if (option.selected) locationValue = `${option.value.trim()}+++`;
+      });
+      locationData[locationProp] = locationValue;
+      if (locationSelect) {
```

Comment:
Given the check on line 83, is this necessary?

Reply:
Thanks for noticing that. Fixed now.